exercises/reverse-string/reverse_string_test.go

```diff
@@ -23,3 +23,11 @@ func TestReverseOfReverse(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+func BenchmarkReverse(b *testing.B) {
+	for _, test := range testCases {
+		for i := 0; i < b.N; i++ {
```

Comment:
I think we should switch order for loops because one of the implementations can be optimized for shorter strings and case might be short then as a result, it might run more and vice versa.

Reply:
I hadn't considered that. So essentially, include all the test cases inside the benchmark loop to avoid optimising for a specific implementation. I'll do that now.